src/skills-builder/skills-builder-modal/view-results/RelatedSkillsSingleBoxSet.jsx

```diff
@@ -0,0 +1,108 @@
+import React, { useRef } from 'react';
+import PropTypes from 'prop-types';
+import {
+  SelectableBox, Chip, Stack, Card, Container,
+} from '@edx/paragon';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import { useVisibilityFlags } from './data/hooks';
+import messages from './messages';
+
+const RelatedSkillsSingleBoxSet = ({ jobSkillsList, selectedJobTitle, onChange }) => {
+  const { formatMessage } = useIntl();
+  const visibilityFlags = useRef(useVisibilityFlags());
+  const { allowMultipleCareerInterests, showSkillsBox, showSkillsList } = visibilityFlags.current;
+
+  // just a no-op for now.
+  // eslint-disable-next-line no-unused-vars
+  const handleSkillClick = (skill) => {
+    // console.log(skill.name);
+  };
+
+  const renderTopFiveSkills = (skills) => {
+    const topFiveSkills = skills.sort((a, b) => b.significance - a.significance).slice(0, 5);
+    return (
+      topFiveSkills.map(skill => (
+        // eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions
+        <span onClick={() => handleSkillClick(skill)}>
+          <Chip
+            key={skill.external_id}
+            className="chip-max-width"
+          >
+            {skill.name}
+          </Chip>
+        </span>
+      ))
+    );
+  };
+
+  // eslint-disable-next-line no-unused-vars
+  const singleJobCard = (() => {
+    const job = jobSkillsList[0];
+
+    return (
+      showSkillsBox
```

Comment:
When would we be modifying this setting? It would appear to just hide the selectable box when it's off? Since changing the setting doesn't trigger a re-render, when would we want to modify this setting?

Also having the conditional at the beginning of the return statement seems off. We should probably either have a ternary that renders null when `false`, or just use the conditional in a parent statement

Reply:
it's just another experimental feature. Turning it off in the constants makes the results page tighter.
I thought the `variable && (<Component/>) ` idiom as common when there isn't a second value for the ternary.